Btms.Backend.IntegrationTests/Consumers/AmazonQueues/AmazonSnsSqsTests.cs

```diff
@@ -0,0 +1,60 @@
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Btms.Backend.IntegrationTests.Consumers.AmazonQueues;
+
+[Collection("AwsSqsSns")]
+[Trait("Category", "Integration")]
+public class AmazonSnsSqsTests : IAsyncLifetime
+{
+    private readonly TestAwsConsumers _awsConsumers = new();
+    private readonly TestAwsSender _awsSender;
+    private readonly ITestOutputHelper _testOutputHelper;
+
+    public AmazonSnsSqsTests(ITestOutputHelper testOutputHelper)
+    {
+        _testOutputHelper = testOutputHelper;
+        _awsSender = new TestAwsSender(_awsConsumers.Configuration, _awsConsumers.AwsSqsOptions, _testOutputHelper);
+    }
+
+    [Fact]
+    public async Task When_checking_if_localstack_available_Then_should_be_available_in_development()
```

Comment:
To date we've used pascal casing here... WhenCheckingIfLocalstackAvailableThenShouldBeAvailableInDevelopment